yarn.lock

```diff
@@ -2690,7 +2690,11 @@
     "@leafygreen-ui/popover" "^7.1.0"
     react-transition-group "^4.4.1"
 
+<<<<<<< HEAD
```

Comment:
It looks like you accidentally committed some merge markers

Reply:
strange, thanks!